docs/_posts/2019-03-05-go.md

```diff
@@ -0,0 +1,50 @@
+
+Go's high-level APIs have also been valuable to running faster development cycles. Usually in a low-level language, it can be a real pain to write hundreds of lines of code just to implement a certain function. With the high-level APIs that Go provides, most of this is concealed through abstraction.  
+
+## The toolchain 
+
+Go’s open source tools and built-in toolchain add additional functionality to projects. These tools can either be imported or implemented into any code environment as stand-alone console applications. Specifically, for the Secretless Broker, the tools we enjoy and find most helpful are `pprof` for profiling, `golint` for code clean up, and `godocs` for documentation automation. 
+
+`pprof`is a profiling tool that once added to the main function of a program, provides an in-depth analysis of processes. With Secretless, every byte of data travels through the broker before reaching and returning to its endpoints. We found that this exchange was taking too long, but we couldn’t tell exactly where the latency was coming from. With pprof, we were able visualize the effect of our project on a host machine’s CPU and memory. This is crucial because by using this tool, we gain visibility into processes and can tune them to provide a more positive experience for our customers. 
+
+It also became difficult to consistently format our code, especially as our codebase increased in size and complexity. To fix this common problem, we added the `golint` automated tool into our workflow. Code linters run source code against an applied set of stylistic and syntax rules, raising flags when this structure is broken.  Linting has been an indispensable productivity tool for writing code and has even sped up our code review process.  
+
+Documentation is essential to growing and sustaining the community around our project. We add to and tweak our README and contributor guides to encourage end-user self-efficiency and success. This has proven to be a time-consuming undertaking because our codebase is constantly evolving and being refactored. To our benefit, Go offers the `godoc` documentation tool to dynamically create documentation. This tool parses the code, relevant comments, and structure to generate documentation that evolves as fast as our code does. 
+
+## … and beyond the toolchain 
+
+With Go, we can produce statically linked binaries that fully contain our applications and run them on different platforms. This capability supported our decision to move to Docker-based development for our building and testing.  
```

Comment:
I don't think I follow the logic here, because the previous sentence makes it sound like you don't need Docker

> This capability supported our decision to move to Docker-based development for our building and testing

Maybe this is a good place to mention that in the early stages of our project we're focusing on getting Secretless working well in Kubernetes and OpenShift, so we wanted to supply our solution in a secure container - but that's only part of the reason we use Docker.

Reply:
Which sentence makes it sound like we don't need docker? About running binaries on different platforms? Adding mention that we wanted to get into K8s and Openshift to further strengthen the initial argument and drive to move to Docker

Reply:
see below @izgeri

Reply:
see below changes